content/en/blog/2024/inpod-traffic-redirection-ambient/index.md

```diff
@@ -0,0 +1,217 @@
+
+## How did we get here?
+
+### Service meshes and CNIs: it's complicated
+
+Istio is a service mesh, and all service meshes by strict definition are not *CNI implementations* - service meshes require a [spec-compliant, primary CNI implementation](https://www.cni.dev/docs/spec/#overview-1) to be present in every Kubernetes cluster, and rest on top of that. 
+
+This primary CNI implementation may be provided by your cloud provider (AKS, GKE, and EKS all ship their own), or by third-party CNI implementations like Calico. Some service meshes may also ship bundled with their own primary CNI implementation, which they explicitly require to function. 
+
+Basically, before you can do things like secure pod traffic with mTLS and apply of high-level authentication and authorization policy at the service mesh layer, you must have a functional Kubernetes cluster with a functional CNI implementation, to make sure the basic networking pathways are set up so that packets can get from one pod to another (and from one node to another) in your cluster.
+
+Though some service meshes may also ship and require their own in-house primary CNI implementation, and it is sometimes possible to run two primary CNI implementations in parallel within the same cluster (for instance, one shipped by the cloud provider, and a 3rd-party implementation), in practice this introduces a whole host of compatibility issues, strange behaviors, reduced featuresets, and some incompatibilities due to the wildly varying mechanisms each CNI implementation might employ internally.
+
+To avoid this, the Istio project has chosen not to ship or require their own primary CNI implementation, or even require a "preferred" CNI implementation - instead choosing to support CNI chaining with the widest possible ecosystem of CNI implementations, and ensuring maximum compatibility with managed offerings, cross-vendor support, and composability with the broader CNCF ecosystem.
+
+### Traffic redirection in ambient alpha
+
+The [istio-cni](/docs/setup/additional-setup/cni/) component is an optional component in the sidecar data plane mode,
+commonly used to remove the [requirement for the `NET_ADMIN` and `NET_RAW` capabilities](/docs/ops/deployment/requirements/) for
+users deploying pods into the mesh. 
+
+istio-cni is a required component in the ambient
+data plane mode. Whenever pods are added to an ambient mesh, the istio-cni component configures traffic redirection for all
+incoming and outgoing traffic between the pods and the [ztunnel](/blog/2023/rust-based-ztunnel/) running on
+the pod's node, via the node-level network namespace. The key difference between the sidecar mechanism and the ambient alpha mechanism is that in the latter, pod traffic was redirected out of the pod network namespace, and into the ztunnel network namespace - necessarily passing through the node's network namespace on the way, which is where the bulk of the traffic redirection rules to achieve this were implemented.
```

Comment:
```suggestion
the pod's node, via the node-level network namespace. The key difference between the sidecar mechanism and the ambient mechanism is that in the latter, pod traffic was redirected out of the pod network namespace, and into the ztunnel network namespace - necessarily passing through the node's network namespace on the way, which is where the bulk of the traffic redirection rules to achieve this were implemented.
```

Reply:
yep agreed, i'd prefer to keep alpha in there to be clear it is the old model.